gha.env

```
@@ -0,0 +1,7 @@
+DOCKERHUB_ORG_NAME=dockerdevrel
```

```suggestion
DOCKERHUB_NAMESPACE=dockerdevrel
```

The thought is that this demo could be used by someone only on a personal plan.